### DISCUSSION TITLE
what happened

### DISCUSSION THREAD

**Discussion by imlens** [Scratch-Bookmarklets/Scratch-Bookmarklets.github.io#195] [General]
the repo just suddenly when dark like no one is doing anything

**Comment by imlens** [comment]
@scratch-bookmarklets

**Comment by Shluffy** [comment]
what do you mean? i think less of us are active

**Comment by imlens** [comment]
ye, and i think thats a problem. We need more OS helpers (open source on operating system)

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
